ui/ozone/platform/wayland/fake_server.cc

```diff
@@ -109,6 +111,39 @@ void CreatePipe(base::ScopedFD* read_pipe, base::ScopedFD* write_pipe) {
   write_pipe->reset(raw_pipe[1]);
 }
 
+void MockZwpLinuxBufferParamsV1_Destroy(wl_client* client,
+                                        wl_resource* resource) {}
```

Comment:
You could call mock methods and verify the parameters are corrected when running tests.